src/actions/actions.ts

```diff
@@ -2112,8 +2112,8 @@ class CommandGoToDefinition extends BaseCommand {
 
 @RegisterAction
 class CommandInsertAtFirstCharacter extends BaseCommand {
-  modes = [ModeName.Normal];
-  mustBeFirstKey = true;
+  modes = [ModeName.Normal, ModeName.Visual];
+  mustBeFirstKey = false;
```

Comment:
Hmm, yeah, that's a good point. Plus, I can't think of any actions that have `A` in the middle of them.